src/Jackalope/Tools/Console/Command/InitDoctrineDbalCommand.php

```diff
@@ -58,22 +67,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
             throw new \InvalidArgumentException('The provided connection is not an instance of the Doctrine DBAL connection.');
         }
 
-        if (true !== $input->getOption('dump-sql')) {
-            $output->write('ATTENTION: This operation should not be executed in a production environment.' . PHP_EOL . PHP_EOL);
+        if (true !== $input->getOption('dump-sql') && !$input->getOption('force')) {
+            $output->write('ATTENTION: This operation should not be executed in a production environment. Please use "--force" to execute the command.' . PHP_EOL . PHP_EOL);
+            return self::RETURN_CODE_NO_FORCE;
         }
 
         $schema = new RepositorySchema;
+
         try {
             if ($input->getOption('drop')) {
                 foreach ($schema->toDropSql($connection->getDatabasePlatform()) as $sql) {
                     if (true === $input->getOption('dump-sql')) {
                         $output->writeln($sql);
                     } else {
-                        try {
-                            $connection->exec($sql);
-                        } catch (\Exception $e) {
-                            $output->writeln($e->getMessage());
-                        }
```

Comment:
It's a bit harder than that, as not all backends use the same error codes IIRC. This has been fixed in dbal 2.5, by throwing very specific exceptions. This way, we can catch only `TableNotFoundException`. However, we also need to support dbal 2.4, which throws the generic `DBALException`. Maybe we can do some string matching?

The current behaviour on jackalope-doctrine-dbal master is: Fail if table to drop did not exists, unless `--force` was passed to the command. I think that's the best option to apply to 1.1 too